packages/tsconfig/react-library.json

```diff
@@ -3,8 +3,9 @@
   "display": "React Library",
   "extends": "./base.json",
   "compilerOptions": {
-    "jsx": "react-jsx",
-    "lib": ["ES2022"],
-    "module": "ES2022"
+    "lib": ["ES2015"],
```

Comment:
same as with nextjs, it makes more sense to be to use ES2022 instead of ESNext or older